templates/docs/patterns/notification.md

```diff
@@ -10,6 +10,10 @@ context:
 
 Notifications are used to attract the user's attention. They offer four separate severity levels, and can also be modified for use in different contexts.
 
+## Live Demo Box
+
+<div id="react-live-demo-box"></div>
```

Comment:
I know it's just a setup PR, so probably something to keep in mind later - we need to be able to have multiple live demo boxes on one page, so the script will have to run on multiple elements (like via "js-example" class name), not just one by id.

Reply:
Good point, we'll keep this in mind :+1: